lua/lazyvim/util/init.lua

```diff
@@ -200,6 +200,18 @@ function M.toggle_number()
   end
 end
 
+function M.toggle_autopairs()
+  if package.loaded["mini.pairs"] then
```

Comment:
But this way, the key/mapping is still set even if the plugin is disabled. I figured by guarding the mapping, the mapping wouldn't be created and it's unlikely the toggle function is called, unless explicitly called.

Reply:
Ah yes, I will move it to the keymap.

Would it be better to use `package.loaded["mini.pairs"]` or `Util.has("mini.pairs") `?

Reply:
I think in `keymaps.lua`, better to use `Util.has` like the other plugin checks in that file.